ui/app/src/state/epics/pathNavigatorTree.ts

```diff
@@ -210,9 +210,9 @@ export default [
           optionsByPath = {
             ...optionsByPath,
             ...createPresenceTable(expanded, (value) => ({
+              ...createGetChildrenOptions(chunk, pluckProps(payload, true, 'limit', 'excludes')),
```

Comment:
Those are the global options for children, they need to go before the specific options per path